### PR TITLE
Fixes #37872 - set windows icon

### PR DESCRIPTION
**Pull request by sbernhard**
Due to license restrictions, we can not use the original windows icons.
Therefore, this icon is based on the windows icon from https://iconoir.com/
It's Open Source:
https://github.com/iconoir-icons/iconoir/blob/main/LICENSE

Original icon:
https://github.com/iconoir-icons/iconoir/blob/f7395255f4621c83f8fa223915e90d36acf38adf/icons/regular/windows.svg?plain=1

### CODE REVIEW

**Comment by ekohl** [comment]
What is the license on something AI generated?

**Comment by sbernhard** [comment]
> What is the license on something AI generated?

According to https://bigsea.co/ideas/bing-image-creator/ and https://www.bing.com/new/termsofuseimagecreator#content-policy yes

**Comment by ekohl** [comment]
I think using AI for this is questionable. Not only from a license perspective. If it's too close, it might still violate the trademark and if it's not close enough it only confuses people. Makes me wonder what other people think about it.

**Comment by evgeni** [comment]
Looking at https://bigsea.co/ideas/bing-image-creator/#image-creator-faqs, it says "Certain limitations exist, such as restrictions on using images containing identifiable people, trademarks, or copyrighted material."
And https://www.bing.com/new/termsofuseimagecreator#content-policy says "Not to infringe on the rights of others. Do not attempt to use Image Creator to infringe on others' legal rights, including intellectual property rights."

I'd argue this is exactly what we have here and thus :-1:

**Comment by sbernhard** [comment]
Understand. 

2 possibilities:

1. thenoundproject:
What about using this: https://thenounproject.com/icon/windows-4796446/
License is here: https://thenounproject.com/legal/terms-of-use/#icon-licenses

As I want to get this done, I would purchase the icon and pay the bill :-) 
image

2. freepik
https://www.freepik.com/icon/windows_5969368#fromView=search&page=1&position=49&uuid=fec914fb-230f-4045-a10f-99fb84b04141

With this, we would need to add a note about the icon somewhere:
https://support.freepik.com/s/article/Attribution-How-when-and-where?language=en_US&_gl=1*1ohif33*_gcl_au*OTY4MzkzNjMuMTczMDgwOTIxMA..*_ga*MjAwMzcxNzEyLjE3MzA4MDkyMTE.*_ga_QWX66025LC*MTczMDgwOTIxMC4xLjEuMTczMDgwOTc0MS4xLjAuMA..*_ga_Q29FZ8F7H4*MTczMDgwOTY2OC4xLjAuMTczMDgwOTY4MC4wLjAuMA..

Opinions?

**Comment by evgeni** [comment]
"no edits" means it's not FOSS. We can't add non-FOSS licensed stuff here.

**Comment by sbernhard** [comment]
> "no edits" means it's not FOSS.

well, you should not edit the debian logo, too :)

**Comment by ekohl** [comment]
I think this is primarily a trademark issue, not a copyright or license.

https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks states:

> Without a written license or express permission under one of Microsoft's other published guidelines (e.g., Specific Brand and Product Guidelines below), don’t do the following:

> Don’t use Microsoft’s logos, icons, or designs, in any manner.

So I think what the Noun Project is doing is legally questionable. They're selling something that another company has a trademark on.

https://cdn-dynmedia-1.microsoft.com/is/content/microsoftcorp/microsoft/mscle/documents/presentations/Windows_Trademark_guidelines_2023.pdf has specific rules for Windows.

In short, :-1: on this unless we get a specific approval from Microsoft to use their symbol.

**Comment by evgeni** [comment]
> well, you should not edit the debian logo, too :)

Yes, that'd be a trademark violation. But the license allows me to (and I can do conversions etc, that don't alter the trademark).

> In short, 👎 on this unless we get a specific approval from Microsoft to use their symbol.

I poked someone we all know at MSFT if there is a nice short cut to get such an approval :)

**Comment by sbernhard** [comment]
> In short, 👎 on this unless we get a specific approval from Microsoft to use their symbol.

Understand. We can not re-use the MS trademark in a different way. 
https://www.freepik.com/icon/grid_3603535#fromView=search&page=2&position=52&uuid=08e76ca5-5975-4a64-92b6-da12f97c1900 this would be a "window", too. 

Would this be acceptable?

**Comment by sbernhard** [comment]
Next try.
This is how it would look like:

image

**Comment by sbernhard** [comment]
@ekohl / @evgeni ?

**Comment by ekohl** [comment]
> I poked someone we all know at MSFT if there is a nice short cut to get such an approval :)

Any news on this?

**Comment by evgeni** [comment]
> > I poked someone we all know at MSFT if there is a nice short cut to get such an approval :)
> 
> Any news on this?

Nope

**Comment by sbernhard** [comment]
Any news @ekohl  / @evgeni ? Do you really think, MS would blame us for this completely different looking window?

**Comment by ekohl** [comment]
I don't want to risk it and when it comes to trademarks. If you feel strongly we need a logo for Windows then please also try to engage with MS directly. Otherwise I'd lean to closing this
